.vscode/launch.json

```diff
@@ -24,6 +24,7 @@
 			"request": "launch",
 			"runtimeExecutable": "${execPath}",
 			"args": [
+				"--disable-extensions",
```

Comment:
Is this to disable other extensions other than openwhisk-extension?

Reply:
Yes, I'll add an annotation for this.
https://code.visualstudio.com/api/working-with-extensions/testing-extension#disabling-other-extensions-while-debugging